src/main/kotlin/com/cognifide/gradle/aem/instance/action/AwaitAction.kt

```diff
@@ -120,7 +120,7 @@ open class AwaitAction(project: Project, val instances: List<Instance>) : Abstra
 
                 // End if assurance is not configured or this moment remains a little longer
                 if (fast || (stableAssurances <= 0) || (sinceStableTicks >= 0 && (timer.ticks - sinceStableTicks) >= stableAssurances)) {
-                    notify("Instance(s) stable", "Which: ${instances.names}")
+                    notify("Instance(s) stable", "Which: ${instances.names}", fast)
```

Comment:
this will cause that there will be no longer displayed `Instances stable` then `Instances healthy` after each other immediatelly, only stable when fast mode is enabled or only healthy when fast mode is disabled (default case).